### PR TITLE
Update quickstart.md -- add sign up link.

### PR DESCRIPTION
**Pull request by ashayh**
Add sign up link.

Description of your changes:

Which issue is resolved by this Pull Request:
Resolves #

Checklist:
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.

[skip ci]

### CODE REVIEW

File: Documentation/quickstart.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ weight: 2
 # Quickstart Guide
 
 Welcome to Rook! We hope you have a great experience installing the Rook storage platform to enable highly available, durable storage
-in your cluster. If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://Rook-io.slack.com).
+in your cluster. If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://Rook-io.slack.com). Sign up for slack [here](https://rook-slackin.herokuapp.com/).
```
Comment (galexrt) [inline Documentation/quickstart.md]: Could you make the `Rook.io` to `rook.io` while you are at it.
I think this `Sign up for our [Slack](https://rook-slackin.herokuapp.com).` makes it sound better.

File: Documentation/quickstart.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ weight: 2
 # Quickstart Guide
 
 Welcome to Rook! We hope you have a great experience installing the Rook storage platform to enable highly available, durable storage
-in your cluster. If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://Rook-io.slack.com).
+in your cluster. If you have any questions along the way, please don't hesitate to ask us in our [Slack channel](https://rook-io.slack.com). Sign up for our slack [here](https://rook-slackin.herokuapp.com/).
```
Comment (galexrt) [inline Documentation/quickstart.md]: Could you make the first letter in `slack` uppercase.

**Comment by travisn** [comment]
@ashayh thanks for contributing, any questions we can help with to get this in?

**Comment by galexrt** [comment]
@ashayh are you available to finish this? If not please let us know and I'll pick it up.

**Comment by ashayh** [comment]
Hi, I added the signed off message and did a force push: https://github.com/ashayh/rook/commit/fa352185a05a9617cafe17a4c50832e487636d7f

Please pick it up if you need to. 
We will be looking at Ceph/rook again, but we are prototyping with NFS right now to get things going.

**Comment by galexrt** [comment]
@ashayh You have two commits now.
You would need to make it into one commit (squash) and then have the sign-off at that commit.

Can you please do that and implement the comment I added in my review?

**Pull request closed without merging**